sdks/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
2019 in Copyright lines

Reply:
Probably we should patch all sources with such headers? Not only new one.

Reply:
Only new files need the new date. The date only needs to be applied at file creation.